core/utils.go

```diff
@@ -1 +1,80 @@
 package core
+
+import (
+	"fmt"
+	"github.com/refractionPOINT/go-limacharlie/limacharlie"
+	"path"
+	"strings"
+	"sync"
+)
+
+var (
+	secretCache = make(map[string]string)
+	cacheMutex  sync.RWMutex
+)
+
+func usingSecretValue(key string, org *limacharlie.Organization, fn func(val string) error) error {
+	var err error
+	var apiKey string
+	var exists bool
+	if strings.Contains(key, "hive://") {
```

Comment:
```suggestion
	if strings.Contains(key, "hive://secret/") {
```
Just slightly deeper validation since we only support secrets anyway.

Reply:
Ah yeah forgot about the secret part.